test/integration/model/bulk-create/include.test.js

```diff
@@ -621,6 +621,119 @@ describe(Support.getTestDialectTeaser('Model'), () => {
         expect(persistedUsers[1].jobs).to.be.ok;
         expect(persistedUsers[1].jobs.length).to.equal(2);
       });
+
+      it('should pass options to associated models as well', async function () {
+        const Player = this.sequelize.define('Player', {
+          code: {
+            type: DataTypes.STRING,
+            primaryKey: true,
+          },
+          name: DataTypes.STRING,
+          surname: DataTypes.STRING,
+          dateBirth: DataTypes.DATE,
+          country: DataTypes.STRING,
```

Comment:
I think it's enough to have code + name. I don't think surname, dob & country are relevant to this test